svc/WebSocketService.js

```diff
@@ -269,6 +273,24 @@ export class WebSocketService extends HoistService {
         });
     }
 
+    async forceAppReloadAsync() {
+        // Suspend activity without setting AppState.SUSPENDED (i.e. don't show Idle panel backdrop)
```

Comment:
Like the idea of putting the user in to the suspended state, but I think for state consistency, would like this to call `XH.idleService.suspendApp().` and have the app *actually* be in a suspended state.   Can we modify that state if needed to display alternate message text, if the suspension was forced?  We could add a method, or flags to suspendApp()?

Reply:
I couldn't find a clear path to switch on SUSPENDED state flavors, so I created a new FORCE_RESTART state and mirrored the implementation of SUSPENDED --> idlePanelHost so that Apps could customize the forceRestartPanel to their liking.

Reply:
discussed with Petra and am looking at a few more changes now to push this over the finish line.